### ISSUE TITLE
Safe mode

### ISSUE THREAD

**Issue by erlend-sh** [kitsune-soc/kitsune#234]
https://calckey.social/notes/9fgpezsdfj6c2ur8

> The fediverse server I want:: 
> 
> - strict content moderation, with focus on providing a safe and welcoming space
> - software allows for granular comment control
> - default setting for comments is 'safe', meaning only people who you follow, and all other people on the same server can comment on your post
> - monthly subscription encouraged, with good part of the money going to professional Trust and Safety team

Could this be supported as a user setting? Personally I’d like to allow comments from everyone, but the above sounds like a sensible configuration for many.

**Comment by aumetra** [comment]
Yeah, that sounds like something that could be supported on a per-user level.  
From the description it sounds like an extension to what GoToSocial allows you do do: https://github.com/superseriousbusiness/gotosocial#granular-post-settings

A relevant thread to some thing relevant to this feature on the SocialHub: https://socialhub.activitypub.rocks/t/fep-5624-per-object-reply-control-policies/2723

---

It would be great if there would be a mutual agreement on how to federate these policies, but for now we can take inspiration from FEP-5624, which is the closest to a standard we have so far.  
We should also take a look into how (if at all) GoToSocial federates these granular post settings, to have some base compatibility with them as well.

But I'd generally be favouring the FEP over an inofficial extension
